clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -9,9 +9,14 @@ export const TimeSlotSelect = () => {
   const [providerTimeSlots, setProviderTimeSlots] = useState<ParsedSlotsType[]>([])
 
   const addTimeSlots = useCallback((newProviderTimeSlots: ParsedSlotsType[]) => {
+    const earliestAvailable = new Date();
+    //appointments within one hour can not be booked
+    earliestAvailable.setTime(earliestAvailable.getTime() + (60*60*1000));
+
     const mergedProviderAvailability = newProviderTimeSlots.map((newProvider) => {
       const provider = providerTimeSlots.find((entry) => entry.providerId === newProvider.providerId)
-      const mergedSlots = provider ? [...provider.providerSlots, ...newProvider.providerSlots] : newProvider.providerSlots
+      const availableSlots = newProvider.providerSlots.filter(slot => new Date(slot.start) > earliestAvailable)
```

Comment:
Should this line use a greater than or equal to check? It's probably unlikely to happen, but there's a possibility that the earliest available date exactly equals a time slot received from the API

```
slot => new Date(slot.start) >= earliestAvailable
```